packages/gitbook-docs/src/markdoc/attributes.ts

```diff
@@ -53,7 +82,42 @@ export class LinkAttr {
             spacePrefix.pathPrefix
           )
         : value;
-    } else return value;
+    } else if (value) {
```

Comment:
Maybe here can be process only `value` that include `DOCS_URL` or is necessary to parse all links ? 
Something like: 
```
} else if (value.includes(DOCS_URL)) {
```